synapse/lib/stormtypes.py

```diff
@@ -6143,6 +6143,9 @@ class View(Prim):
                 nomerge (bool)
                     Setting to $lib.true will prevent the layer from being merged.
 
+                layers (list(str))
+                    Set the list of layer idens for a non-forked view.
```

Comment:
Do we want to note the layers are set top down such that layers[0] is the write layer?